NUnitLite/TouchRunner/TouchOptions.cs

```diff
@@ -95,6 +98,7 @@ public TouchOptions ()
 				{ "autostart", "If the app should automatically start running the tests.", v => AutoStart = true },
 				{ "hostname=", "Comma-separated list of host names or IP address to (try to) connect to", v => HostName = v },
 				{ "hostport=", "HTTP/TCP port to connect to.", v => HostPort = int.Parse (v) },
+				{ "use_tcp_tunnel", "Use a tcp tunnel to connect to the host.", v => UseTcpTunnel = true },
```

Comment:
other options don't use underscore in their names 
TCP should be capitalized in the description